.github/actions/build-setup/action.yml

```diff
@@ -4,6 +4,32 @@ description: "Setup a common environment for CI jobs"
 runs:
   using: "composite"
   steps:
+    - name: CPUInfo
+      shell: bash
+      run: |
+        cat /proc/cpuinfo
+    - name: MemInfo
+      shell: bash
+      run: |
+        cat /proc/meminfo
+    - name: install common dependencies
+      shell: bash
+      run: |
+        # prepare new user's home directory with symlink, mv takes a full minute.
```

Comment:
This changes makes prepare take 12 seconds vrs 3 minutes.

Reply:
This step should be moved below the one that actually runs the install script and they currently have the same name, either merge the steps or come up with a new name for this step.

Reply:
It can't be unless we want builds to  take 3 minutes longer.   Calls for more info in the doc as to whats going on if you thought this could/needed to be moved.  This changes heavily for sccache to work (and files to move locations).   I'll follow up in the other pr.